docs/api.rst

```diff
@@ -3019,9 +3039,9 @@ Adding custom ethereum tokens
           "cryptocompare": "FOO",
           "protocol": "uniswap",
           "underlying_tokens": [
-              {"address": "0x4a363BDcF9C139c0B77d929C8c8c5f971a38490c", "weight": "15.45"},
-                  {"address": "0xf627B24754583896AbB6376b1e231A3B26d86c99", "weight": "35.65"},
-                  {"address": "0x2B18982803EF09529406e738f344A0c1A54fA1EB", "weight": "39"}
+              {"address": "0x4a363BDcF9C139c0B77d929C8c8c5f971a38490c", "chain":"ethereum", "token_kind":"erc20", "weight": "15.45"},
```

Comment:
Yes, chain would be the same always. The reason to add it was for easy access without the need of getting the parent token since it was done one place and saw it useful. But we can just remove this attribute and get it from the parent